api/v01/entities/vrp_input.rb

```diff
@@ -591,7 +591,7 @@ module VrpVehicles
 
   params :router_options do
     optional :router_mode, type: String, desc: '`car`, `truck`, `bicycle`, etc... See the Router Wrapper API doc.'
-    exactly_one_of :matrix_id, :router_mode
+    at_least_one_of :matrix_id, :router_mode
```

Comment:
Is there a  use case for a user to have both of these params?

Reply:
It could happen if a user transmits its own matrix, but still want the route polylines to be returned

Reply:
Then we might want to do `allow_blank: false` for both of these,  no? Otherwise `nil` `nil` for both passes this verification